modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1965,7 +1956,7 @@ def astype(self, col_dtypes, errors: str = "raise"):
         # invalid type keys.
         return self.__constructor__(self._modin_frame.astype(col_dtypes, errors=errors))
 
-    def infer_objects(self):
+    def infer_objects(self, copy):
```

Comment:
Why is `copy` not propagated into the method of the `modin_frame`?

Reply:
https://github.com/modin-project/modin/issues/6216